ts/src/pro/coinex.ts

```diff
@@ -430,25 +547,28 @@ export default class coinex extends coinexRest {
         /**
          * @method
          * @name coinex#watchTickers
-         * @see https://viabtc.github.io/coinex_api_en_doc/spot/#docsspot004_websocket007_state_subscribe
          * @description watches a price ticker, a statistical calculation with the information calculated over the past 24 hours for all markets of a specific list
+         * @see https://docs.coinex.com/api/v2/spot/market/ws/market
+         * @see https://docs.coinex.com/api/v2/futures/market/ws/market-state
          * @param {string[]} symbols unified symbol of the market to fetch the ticker for
          * @param {object} [params] extra parameters specific to the exchange API endpoint
          * @returns {object} a dictionary of [ticker structures]{@link https://docs.ccxt.com/#/?id=ticker-structure}
          */
         await this.loadMarkets ();
-        symbols = this.marketSymbols (symbols);
+        let marketIds = this.marketIds (symbols);
         let type = undefined;
         [ type, params ] = this.handleMarketTypeAndParams ('watchTickers', undefined, params);
         const url = this.urls['api']['ws'][type];
         let messageHash = 'tickers';
-        if (symbols !== undefined) {
+        if (marketIds !== undefined) {
             messageHash = 'tickers::' + symbols.join (',');
```

Comment:
watchTickers should use the `watchMultiple` approach now (instead of using a single messageHash), please check other implementations